gnomad_qc/v4/create_release/create_release_sites_ht_exomes.py

```diff
@@ -64,7 +68,7 @@
     "region_flags",
     "in_silico",
     "vep",
-    # "joint_faf",
+    "joint_faf",
```

Comment:
another global I realized was missing from this HT while documenting the schema on the help/faq page is a potential readme describing how to use the frequency fields (this was proposed for v3) -- is this something we still want to add, or will we point users to the instructions on the help page?

Reply:
Ah right, Dan King asked for that right? I think it makes sense to include if that got all the way back to us from hail users. It's not super intuitive. I'll draft something.

Reply:
wait, isn't that here: https://gnomad.broadinstitute.org/help#frequently-asked-questions under How do I access the gnomAD Hail Table frequency annotation

Reply:
It think we just point there right?

Reply:
Maybe put that as a main topic not just a frequently asked question?

Reply:
yes, it's on the help page, but it doesn't hurt to add to the HT itself as well, right?

here's Dan's initial request
>Random thought, maybe you can include a README global field in the next release with just some text describing where to get more information about the meaning of adj, raw, what the freq fields are, etc. Or maybe there could be a more visible link to the Help section right about the sites table? I'm not exactly sure what the issue is but maybe a couple times a year someone shows up to the Hail forum flummoxed by the freq array

Reply:
Added and pointed to the FAQ https://github.com/broadinstitute/gnomad_qc/pull/477/commits/b3b9fcefc0841a2cce618b39b5cb98138125c70d